### DISCUSSION TITLE
Converting WGS84 to UTM coordinates

### DISCUSSION THREAD

**Discussion by pfrollo** [pyproj4/pyproj#1079] [Q&A] [answered]
I am trying to convert WGS84 coordinates to UTM however I am not getting the expected results.

**Input:**

- Lat: -16 degrees
- Lon: -69 degrees 
- UTM zone: 19 (19S)

**Expected output according to [montana.edu](http://rcn.montana.edu/resources/Converter.aspx)**

- Easting: 500000
- Northing: **8231065**

**Received output:**

- Easting: 500000
- Northing: **-1768935**

**Steps to reproduce:** 

```
~ > pip list|grep pyproj
pyproj                            3.3.1
~ > python3
Python 3.9.12 (main, May  8 2022, 18:05:28) 
[Clang 13.0.0 (clang-1300.0.29.30)] on darwin
>>> import pyproj
>>> from pyproj import Proj
>>> p = Proj(proj='utm', zone='19', ellps='WGS84', preserve_units=False)
>>> x, y = p(-69, -16)
>>> print("{} {}".format(x, y))
500000.0 -1768935.375976459
```

 Our projects has hard limits on old versions of Python (3.6.0) and pyproj (2.4.0). I tried the same code in different versions of pyproj with the same results.

Any help is appreciated! Thank you!

**Comment by snowman2** [comment, marked as answer]
You need the `south` indicator:

```python
 p = Proj(proj='utm', zone='19', ellps='WGS84', south=True, preserve_units=False)
```

**Reply by pfrollo** [reply]
Thanks! I owe you!
